docs/how-to/customise-pdf.rst

```diff
@@ -0,0 +1,260 @@
+
+Any additions or changes to the default settings of LaTeX elements in the PDF document requires overriding the default template.
+
+1. Download the default template file `latex_elements_template.txt <https://github.com/canonical/canonical-sphinx/blob/main/canonical_sphinx/theme/PDF/latex_elements_template.txt>`_ from the ``canonical/canonical-sphinx`` GitHub repository, and save it to your documentation project directory. For example, at :file:`.sphinx/latex_elements_custom.txt`.
+
+2. In the Python dictionary, add or modify the LaTeX elements you want to change. Details of changing the dictionary are covered in the sub-sections below.
+
+3. In your project :file:`conf.py`, add or update the ``latex_elements`` dictionary to load the local override of the LaTeX template. Change the file path to the location of your local override file.
+
+.. code-block:: python
+
+   # Replace with the path to your local override file
+   latex_elements_file = ".sphinx/latex_elements_custom.txt"  
+   
+   with open(latex_elements_file, "rt") as file:
+      latex_config = file.read()
+      if latex_elements == {}:
+         latex_elements = ast.literal_eval(latex_config)
+
+.. warning::
+
+   Defining other settings directly in ``latex_elements`` will override the values loaded from the template file or your local file.
+
+
+Add more LaTeX packages to the preamble
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can use two methods to add additional LaTeX packages to the preamble:
+
+* Add the ``extrapackages`` key in your local template file:
+
+  .. code-block:: python
+     :emphasize-lines: 3
+
+     {
+       ...
+       'extrapackages': r'\usepackage{packagename}',
+       ...
+     }
+
+* Modify the values of the ``preamble`` key in your local template file. This is more flexible for adding LaTeX configurations and commands to the preamble.
+
+.. note:: 
+   The format of the element values is a multi-line string, so use a raw string with the ``r`` prefix.
+
+
+Remove the table of contents
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For a short, compact document where navigation is not needed, you may want to remove the table of contents from the PDF output. 
+
+To do this, provide a local copy of the default template file, and add a new key ``tableofcontents`` with an empty string as the value:
+
+.. code-block:: python
+   :emphasize-lines: 3
+
+   {
+      ...
+      'tableofcontents': '',
+      ...
+   }
+   
+
+Include images or other assets
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If the local template requires additional images or other assets, for example, a custom title page or header, the file paths must be added to the Sphinx :file:`conf.py` file to be included in the PDF build.
+
+Provide a ``latex_additional_files`` variable in :file:`conf.py` as a list of file paths to the additional assets. If the variable already exists, add the new file paths to the list.
```

Comment:
It's probably worth adding here that vector file types are recommended (pdf/png) and which others are supported.